django-backend/fecfiler/contacts/views.py

```diff
@@ -32,6 +33,14 @@
 )
 
 
+def validate_candidate(candidate_id):
+    return len(candidate_id) == 9
+
+
+def sanitize(candidate_id):
+    return candidate_id
```

Comment:
Perhaps to sanitize, we could check for valid characters. I believe it's just numbers and a handful of capital letters. Let's check with Gregg.